tests/cypress/support/index.js

```diff
@@ -22,6 +21,7 @@ before(() => {
 // Chrome: ResizeObserver loop limit exceeded
 // Firefox: ResizeObserver loop completed with undelivered notifications
 const resizeObserverLoopErr = 'ResizeObserver loop';
+// eslint-disable-next-line consistent-return
```

Comment:
Maybe ``return true;`` is better at the end of the listener?

Reply:
Yes, it is better. Thanks! Fixed.